scss/_patterns_chip.scss

```diff
@@ -64,234 +65,204 @@ $chip-border-thickness: 1px;
-    color: $color-chip-lead;
+
+  // stylelint-disable-next-line selector-max-type
+  body.is-dark .p-chip--negative .p-chip__dismiss {
+    @include vf-icon-close(#fff);
   }
+}
+
+@mixin vf-chips-information {
+  .p-chip--information {
+    @extend %vf-chip;
+
+    background-color: $colors--theme--background-information-default;
+    border: $chip-border-thickness solid $colors--theme--border-information;
+    border-color: $colors--theme--border-information;
 
-  @if ($is-interactive) {
     &:hover {
-      background-color: $color-background-hover;
-      border-color: $color-border;
+      background-color: $colors--theme--background-information-hover;
+      border-color: $colors--theme--border-information;
     }
+
     &[aria-pressed='true'],
     &.is-selected,
     &:active {
-      background-color: $color-background-active;
-      border-color: $color-border;
-    }
-  } @else {
-    .p-chip__dismiss {
-      @include vf-icon-close($color-chip-lead);
-
-      @include vf-button-pattern(
-        $button-background-color: transparent,
-        $button-border-color: transparent,
-        $button-hover-background-color: $color-background-hover,
-        $button-hover-border-color: transparent,
-        $button-active-background-color: $color-background-active,
-        $button-active-border-color: transparent
-      );
+      background-color: $colors--theme--background-information-active;
+      border-color: $colors--theme--border-information;
     }
   }
-}
 
-@mixin vf-chip-light-theme($chip-type: neutral, $is-interactive: false) {
-  @include vf-chip-theme(
-    $chip-type: $chip-type,
-    $is-interactive: $is-interactive,
-    $colors-chip-tinted-backgrounds: $colors-light-theme--tinted-backgrounds,
-    $colors-chip-tinted-borders: $colors-light-theme--tinted-borders,
-    $color-chip-value: $colors--light-theme--text-default,
-    $color-chip-lead: $colors--light-theme--text-default
-  );
-}
+  .p-chip--information .p-chip__dismiss {
+    @include vf-icon-close(#000);
+    @include vf-button-pattern(
+      $button-background-color: transparent,
+      $button-border-color: transparent,
+      $button-hover-background-color: $colors--theme--background-information-hover,
+      $button-hover-border-color: transparent,
+      $button-active-background-color: $colors--theme--background-information-active,
+      $button-active-border-color: transparent
+    );
+  }
 
-@mixin vf-chip-dark-theme($chip-type: neutral, $is-interactive: false) {
-  @include vf-chip-theme(
-    $chip-type: $chip-type,
-    $is-interactive: $is-interactive,
-    $colors-chip-tinted-backgrounds: $colors-dark-theme--tinted-backgrounds,
-    $colors-chip-tinted-borders: $colors-dark-theme--tinted-borders,
-    $color-chip-value: $colors--dark-theme--text-default,
-    $color-chip-lead: $colors--dark-theme--text-default
-  );
+  // stylelint-disable-next-line selector-max-type
+  body.is-dark .p-chip--information .p-chip__dismiss {
```

Comment:
The dismiss button doesn't seem to depend on the variant of the chip, only on the theme. It's the same in all the variants (within the same theme), so seems redundant to have it defined for all of them separately: `body.is-dark .p-chip--information .p-chip__dismiss` `body.is-dark .p-chip--caution .p-chip__dismiss`, etc.

Would it work if defined in one place as `body.is-dark .p-chip__dismiss` for all the chips?

Reply:
Also, currently it only work with `is-dark` set on body, but we need it to work with `is-dark` set on chip itself.

So it needs to be something like:

```
body.is-dark .p-chip__dismiss,
.p-chip.is-dark .p-chip__dismiss,
[class*='.p-chip--'].is-dark .p-chip__dismiss { // this [class*=] will match all class names containing `p-chip--` which should be all our chips variants
    @include vf-icon-close(#fff);
}